jupyterhub/schema.yaml

```
@@ -1160,6 +1160,43 @@ properties:
             patternProperties: *labels-and-annotations-patternProperties
             description: |
               Kubernetes annotations to apply to the k8s ServiceAccount.
+      extraPodSpec: &jupyterhub-extra-pod-spec
```

```suggestion
      extraPodSpec: &extraPodSpec-spec
```

This aligns with a not so obvious way of naming these anchors for whenever they reference a duplicated configuration. There are other anchors named differently when they are referencing something entirely different etc.